cmd/mirror-main.go

```diff
@@ -303,17 +303,19 @@ func (mj *mirrorJob) doMirror(ctx context.Context, cancelMirror context.CancelFu
 		sURLs.TargetContent.Metadata["X-Amz-Storage-Class"] = mj.storageClass
 	}
 
-	// Set multiMasterETagKey for the target.
-	if sURLs.SourceContent.UserMetadata[multiMasterETagKey] != "" {
-		sURLs.TargetContent.Metadata[multiMasterETagKey] = sURLs.SourceContent.UserMetadata[multiMasterETagKey]
-	} else {
-		sURLs.TargetContent.Metadata[multiMasterETagKey] = sURLs.SourceContent.ETag
-	}
+	if mj.multiMasterEnable {
```

Comment:
perhaps its also `mj.multiMasterEnable || mj.preserve`

Reply:
done

Reply:
if the metadata is already set, it gets copied over to the destination by default. Currently, only if `multi-master` is enabled do we generate the `etag` and `stag` metadata on the destination.